[debug] [2020-04-10T14:35:59.160Z] ----------------------------------------------------------------------
[debug] [2020-04-10T14:35:59.165Z] Command:       /usr/local/bin/node /usr/local/bin/firebase init
[debug] [2020-04-10T14:35:59.165Z] CLI Version:   8.0.2
[debug] [2020-04-10T14:35:59.165Z] Platform:      darwin
[debug] [2020-04-10T14:35:59.166Z] Node Version:  v12.13.1
[debug] [2020-04-10T14:35:59.166Z] Time:          Fri Apr 10 2020 16:35:59 GMT+0200 (Central European Summer Time)
[debug] [2020-04-10T14:35:59.167Z] ----------------------------------------------------------------------
[debug] [2020-04-10T14:35:59.167Z] 
[debug] [2020-04-10T14:35:59.185Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-04-10T14:35:59.267Z] Error: Could not load the default credentials. Browse to https://cloud.google.com/docs/authentication/getting-started for more information.
    at GoogleAuth.getApplicationDefaultAsync (/usr/local/lib/node_modules/firebase-tools/node_modules/google-auth-library/build/src/auth/googleauth.js:160:19)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async GoogleAuth.getClient (/usr/local/lib/node_modules/firebase-tools/node_modules/google-auth-library/build/src/auth/googleauth.js:502:17)
    at async GoogleAuth.getAccessToken (/usr/local/lib/node_modules/firebase-tools/node_modules/google-auth-library/build/src/auth/googleauth.js:524:24)
[error] 
[error] Error: An unexpected error has occurred.
